tools/accuracy/accuracy_checker.py

```diff
@@ -82,6 +82,27 @@ def parse_args():
                         default=False,
                         help='Turn on ort VERBOSE logging via session options')
 
+    parser.add_argument(
+        '--disable-offload-copy',
```

Comment:
offload copy by default is false. this should be enable_offload_copy

Reply:
Its on by default for the python API.

Reply:
Oh right